src/main/resources/static/javascript/mainApp/general.controller.js

```diff
@@ -14,7 +14,7 @@
      */
 
     function GeneralJsController($scope, $controller, $window, $uibModal, groupingsService, dataProvider, Message) {
-        // This is a regex Pattern that contains all valid UH Identifiers which consists of uid (Username) and uhUuid (UH Numbers) chars.
+        // This is a regex Pattern that contains all valid UH Identifiers which consists of uid (Uid) and uhUuid (UH Numbers) chars.
```

Comment:
The `(Uid)` in the comment should stay as `(Username)` because the comment explains what is a uid and uhUuid.